composer.json

```
@@ -25,16 +25,22 @@
             "Ratchet\\": "src/Ratchet"
         }
     }
+   , "autoload-dev": {
+        "psr-4": {
+            "Ratchet\\Tests\\": "tests/"
+        }
+    }
   , "require": {
-        "php": ">=5.4.2"
-      , "ratchet/rfc6455": "^0.3.1"
-      , "react/socket": "^1.0 || ^0.8 || ^0.7 || ^0.6 || ^0.5"
-      , "react/event-loop": ">=0.4"
-      , "guzzlehttp/psr7": "^1.7|^2.0"
-      , "symfony/http-foundation": "^2.6|^3.0|^4.0|^5.0|^6.0"
-      , "symfony/routing": "^2.6|^3.0|^4.0|^5.0|^6.0"
+        "php": "^7.3 || ^8.0"
+      , "ratchet/rfc6455": "^0.3"
+      , "react/socket": "^1.0 || ^0.8 || ^0.7 || ^0.6 || ^0.5 || ^v1.15"
+      , "react/event-loop": ">=v1.5.0"
```

```suggestion
      , "react/socket": "^1.0 || ^0.8 || ^0.7 || ^0.6 || ^0.5"
      , "react/event-loop": ">=0.4"
```
*Those changes are unnecessary because the previous range already allows those newer versions